### PR TITLE
BaSP-TG-27: Compiled Modal/Add&EditForms/Delete feature

### PR DESCRIPTION
**Pull request by JulianFloresDev**
Create Pull request to add shared table component.

Also we add **_delete feature_** to table component.

In addition, we removed the unused files and folders from last week's folder structure.

### CODE REVIEW

File: src/Components/Share/Form/index.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+      console.error(error);
+    }
+  };
+
+  return (
+    <section>
+      {/* <Modal></Modal> */}
+      <form>
+        {properties.map((prop, index) => {
+          console.log(data);
```
Comment (JulianFloresDev) [inline src/Components/Share/Form/index.js:70]: Delete console.logs

File: src/Components/Share/Form/index.js [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+  const entitie = URLPath[1];
+  const properties = Object.keys(data);
+
+  // useEffect(async () => {
+  //   if (id !== '0') {
+  //     const request = id && (await fetch(`${process.env.REACT_APP_API_URL}/${entitie}/${id}`));
+  //     const response = await request.json();
+  //     setData(response.data || {});
+  //   }
+  // }, []);
```
Comment (JulianFloresDev) [inline src/Components/Share/Form/index.js:24]: It should be remove

File: src/Components/Share/Form/index.js [hunk 3]
```diff
@@ -0,0 +1,221 @@
+import { useEffect, useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+// import Modal from '../Modal';
```
Comment (JulianFloresDev) [inline src/Components/Share/Form/index.js:3]: Delete comment lines

File: src/Components/Share/Form/index.js [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+      setProjectsList(dataProjects.data);
+      const resTasks = await fetch(`${process.env.REACT_APP_API_URL}/tasks`);
+      const dataTasks = await resTasks.json();
+      setTasksList(dataTasks.data);
+    } catch (err) {
+      console.error(err);
+    }
+  }, []);
+
+  // const [errorMessage, setErrorMessage] = useState('');
```
Comment (JulianFloresDev) [inline src/Components/Share/Form/index.js:49]: It should be remove

File: src/Components/Share/CreateForm/index.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+        })}
+        <div>
+          <button
+            onClick={(e) => {
+              e.preventDefault();
+              createRow();
+              history.goBack();
+            }}
+          >
+            Submit
```
Comment (JulianFloresDev) [inline src/Components/Share/CreateForm/index.js:98]: I think it should be 'Create'

**Review by DinoStampella — DISMISSED** [review]

**Review by agusgatto8 — APPROVED** [review]

**Review by kolodani — APPROVED** [review]
great job !!!

**Pull request merged**